### ISSUE TITLE
unexpected exit on read_json_auto()

### ISSUE THREAD

**Issue by thbley** [duckdb/duckdb#6432]
### What happens?

Cli crashes when loading very big json files. I'm not sure what's the supported size limits for loading json files, so this could be also a feature request.

### To Reproduce

Test 1:

```
wget -O test.json "https://www.kartenliebe.de/web-api/calendar-data"
ls -lisah test.json
# 2753933 1,3M -rw-rw-r-- 1 tb tb 1,3M Feb 22 21:53 test.json
./duckdb
# v0.7.1-dev318 c4dc292659
# Enter ".help" for usage hints.
# Connected to a transient in-memory database.
# Use ".open FILENAME" to reopen on a persistent database.
select * from read_json_auto('test.json');
# malloc_consolidate(): invalid chunk size
# Aborted
echo $?
# 134
```

Test 2:

```
wget -O test2.json "https://www.kartenliebe.de/web-api/card-data?occasion=kommunion&themeId=2813&card=fotostreifen-90x210"
ls -lisah test2.json
# 2758495 1,3M -rw-rw-r-- 1 tb tb 1,3M Feb 22 21:58 test2.json
./duckdb
# v0.7.1-dev318 c4dc292659
# Enter ".help" for usage hints.
# Connected to a transient in-memory database.
# Use ".open FILENAME" to reopen on a persistent database.
select * from read_json_auto('test2.json');
# Segmentation fault
echo $?
# 139
```

### OS:

x64 amd

### DuckDB Version:

v0.7.1-dev318 c4dc292659

### DuckDB Client:

cli

### Full Name:

Thomas Bley

### Affiliation:

myself

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by lnkuiper** [comment]
Thank you for the test cases. I can confirm this still breaks on the latest master.

**Comment by lnkuiper** [comment]
I have pushed a fix in #6437
